utils/splunk/splunk_telemetry_base.py

```diff
@@ -53,6 +94,9 @@ def check(self, instance):
             # If no service checks were produced, everything is ok
             if len(self.service_checks) is 0:
                 self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK)
+        except TokenExpiredException as e:
+            self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, tags=instance.tags, message=str(e.message))
+            self.log.exception("Splunk topology exception: %s" % str(e.message))
```

Comment:
Out of curiosity, do we also get to see the stack trace upon error? Only a message might not help debugging the code.